mne/preprocessing/ica.py

```diff
@@ -1115,6 +1115,28 @@ def _find_bads_ch(self, inst, chs, threshold=3.0, start=None,
 
         return labels, scores
 
+    def _get_ctps_threshold(self, Pk_threshold=1e-20):
```

Comment:
```suggestion
    def _get_ctps_threshold(self, pk_threshold=1e-20):
```
you should not have capitals in snake_case variable names if you follow pep8 rule.

Reply:
In the paper, Pk was log-transformed to pk: pk = -log10(Pk). Because the threshold value used here was Pk, I wondered if `Pk_threshold` would be more appropriate than `pk_threshold`? or maybe `significance_level` would be better?